root/static/scripts/edit/utility/linkPhrase.js

```diff
@@ -13,7 +13,9 @@ const clean = require('../../common/utility/clean');
 var attributeRegex = /\{(.*?)(?::(.*?))?\}/g;
 
 function mapNameToID(result, info, id) {
-    result[info.attribute.name] = id;
+    if (info.attribute !== undefined) {
```

Comment:
It doesn't make sense for this to be undefined, and will cause things to break elsewhere, so the bug is actually: what is causing this to be undefined?

It looks like the problem is that @reosarevok added a new link attribute type named "number," but there's already one named "number," so it overwrites the original one here: https://github.com/metabrainz/musicbrainz-server/blob/cb20c520de4f197d3b83a8a7624d91b6f88a3b2c/lib/MusicBrainz/Server/Form/Utils.pm#L221

We'll have to see if it's necessary for things to be keyed by name there, or for names to be unique (I believe they've always been up until this point, perhaps 'cause we only expose the name in the web service, and not for any other reason).